spec/models/ability_spec.rb

```diff
@@ -210,7 +208,7 @@
       it{ should_not be_able_to(:manage, my_conference.tickets.first) }
       it{ should_not be_able_to(:manage, conference_public.tickets.first) }
 
-      it{ should_not be_able_to(:manage, my_registration) }
+      it{ should be_able_to(:manage, my_registration) }
```

Comment:
Every signed in user can manage his/her own registration. I think that this is not what this test is testing, because my_registration was a registration for another user. How come you changed the test?